devito/dle/backends/basic.py

```diff
@@ -40,7 +39,7 @@ def _create_efuncs(self, nodes, state):
         """
         noinline = self._compiler_decoration('noinline', c.Comment('noinline?'))
 
-        functions = OrderedDict()
+        efuncs = OrderedDict()
         mapper = {}
         for tree in retrieve_iteration_tree(nodes, mode='superset'):
             # Search an elementizable sub-tree (if any)
```

Comment:
What exactly is meant by the word "elementizable"   ?

Reply:
a very bad name I chose long time ago to indicate functions that are called from within the main kernel body